app/models/work.rb

```diff
@@ -178,7 +178,7 @@ def new_recipients_allow_gifts
       next if gift.pseud&.user&.preference&.allow_gifts?
       next if self.challenge_assignments.map(&:requesting_pseud).include?(gift.pseud)
       next if self.challenge_claims.reject { |c| c.request_prompt.anonymous? }.map(&:requesting_pseud).include?(gift.pseud)
-
+      
```

Comment:
Can you remove this spacing change?

Reply:
Yup! That one crawled in when I wasn't looking. I'll get it changed and push a commit with it fixed.